sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_api_versions.py

```diff
@@ -7,10 +7,8 @@
 from azure.core import CaseInsensitiveEnumMeta
 
 class ApiVersion(str, Enum, metaclass=CaseInsensitiveEnumMeta):
-    V2023_03_06 = "2023-03-06"
     V2023_06_15_PREVIEW = "2023-06-15-preview"
-    V2023_10_15 = "2023-10-15"
     V2023_10_03_PREVIEW = "2023-10-03-preview"
-    V2024_04_15 = "2024-04-15"
+    V2024_06_15_PREVIEW = "2024-06-15-preview"
```

Comment:
why was GA versions removed here?

Reply:
my bad, it shouldn't be added back will push the changes.